### PR TITLE
fix(TMRX-1314): swap horizontal article stack with vertical

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
swap horizontal article stack with vertical

[JIRA-1314](https://nidigitalsolutions.jira.com/browse/TMRX-1314)

### CODE REVIEW

File: packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx [hunk 1]
```diff
@@ -25,26 +25,27 @@ export const ArticleStack = ({
...
     ...item,
     imageTop: true,
     headlineTypographyPreset: 'editorialHeadline020',
     hasTopBorder: false
   }));
 
-  const modifiredHorizontalArticles = horizontalArticles.map(item => ({
+  const modifiedVerticalArticles = verticalArticles.map(item => ({
     ...item,
     imageTop: true,
```
Comment (bibicollins1) [inline packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx:37]: Does this want imageTop still?

Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx:37]: yes it does and the reason for that is default layout of leadArticle is `image right` so when you hideImage it still takes the default layout if `imageTop: true` not provided but I agree if we hide the image we should not need to pass the `imageTop: true` and I will create a tech debt ticket to refactor it

**Review by bibicollins1 — APPROVED** [review]

File: packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx [hunk 2]
```diff
@@ -25,26 +25,27 @@ export const ArticleStack = ({
...
   }));
-  const articleGridVertical = (
+  const articleStackHorizontal = (
     <GridLayout
       columns={{ md: '1px 1fr 1px 1fr' }}
       columnGap={{ md: 'space040' }}
       overrides={{ marginBlockStart: 'space000' }}
     >
-      {modifiedVerticalArticles.map(
+      {modifiredHorizontalArticles.map(
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx:48]: `modifiedHorizontalArticles`? Opposed to `modifiredHorizontalArticles`?

File: packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx [hunk 3]
```diff
@@ -25,26 +25,27 @@ export const ArticleStack = ({
   horizontalArticles: LeadArticleProps[];
   horizontalArticleContentWidth?: string;
 }) => {
-  const modifiedVerticalArticles = verticalArticles.map(item => ({
+  const modifiredHorizontalArticles = horizontalArticles.map(item => ({
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx:28]: `modifiedHorizontalArticles`? Opposed to `modifiredHorizontalArticles`?

**Review by douglasmik — APPROVED** [review]

File: packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx (hunk 2 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx:48]: good catch

File: packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx (hunk 3 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx:28]: you have got good eye

**Review by douglasmik — COMMENTED** [review]
Please resolve requested changes - approved

**Review by yukiinasawa — APPROVED** [review]

**Pull request merged**